pkg/koordlet/runtimehooks/runtimehooks.go

```diff
@@ -87,3 +95,24 @@ func registerPlugins() {
 		klog.Infof("runtime hook plugin %s enable %v", hookFeature, enabled)
 	}
 }
+
+const (
+	defaultRuntimeHookConfigFilePath string = "/etc/runtime/hookserver.d/koordlet.json"
```

Comment:
make defaultRuntimeHookConfigFilePath as an arg, besides, remember to record an issue on charts repo to update the koordlet yaml.